src/utils/roomHelper.ts

```
@@ -15,6 +15,38 @@ export const checkUserInRoom = async (socket: Socket): Promise<{ room: string, r
+    return null;
+  }
+
+  const socketToCheck = socket.id ? socket.id : socket;
+  console.log(`Socket: ${socketToCheck}, Code: ${code}, Role: ${role}`);
+
+  try {
+    if (role === 'god') {
+      console.log("User is a god");
+      const room = await clientDB.collection('rooms').findOne({ code, 'gods.id': socketToCheck });
+
+      if (room && room.gods) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)